src/app/organizations/collections/create-collection/create-collection.component.html

```diff
@@ -13,7 +13,9 @@
    See the License for the specific language governing permissions and
    limitations under the License.
 -->
-<div *ngIf="title$ | async as title">
+<ng-container *ngIf="title$ | async as title">
+<ng-container *ngIf="saveLabel$ | async as saveLabel">
```

Comment:
Shouldn't this be indented as it's a nested block?

Reply:
done